crowbar_framework/app/models/service_object.rb

```diff
@@ -1551,26 +1544,6 @@ def run_remote_chef_client(node, command, logfile_name)
     }
   end
 
-  def chef_daemon(action, node_list)
```

Comment:
are you sure that nothing else uses them? I didn't want to take the risk right now..